### PR TITLE
Server should not ignore plaintext packets as long as it has stuff to rtx

### PR DESCRIPTION
**Pull request by julek-wolfssl**

### CODE REVIEW

**Comment by ejohnstown** [comment]
please retest this

**Comment by ejohnstown** [comment]
retest this please

**Review by ejohnstown — DISMISSED** [review]
The test case I was trying to fix appears to work. Looks good!

File: tests/api.c [hunk 1]
```diff
@@ -64569,6 +64569,73 @@ static int test_dtls_downgrade_scr(void)
 }
 #endif
 
+/* Make sure we don't send acks before getting a server hello */
```
Comment (julek-wolfssl) [inline tests/api.c:64572]: Need to update this comment to reflect that we are testing the changecipherspec message being dropped. @JacobBarthelmeh if you have a moment you can do this and push to my branch to make the merge today.

**Review by JacobBarthelmeh — DISMISSED** [review]

File: tests/api.c (hunk 1 above)
Comment (julek-wolfssl) [reply tests/api.c:64572]: Thanks @JacobBarthelmeh!

**Review by rizlik — DISMISSED** [review]

**Comment by julek-wolfssl** [comment]
Retest this please.

**Comment by bandi13** [comment]
Retest this please.

**Review by ejohnstown — APPROVED** [review]

**Review by JacobBarthelmeh — APPROVED** [review]

**Pull request merged**